.github/workflows/publish_wiki.yaml

```diff
@@ -0,0 +1,51 @@
+name: Publish Wiki
+
+on:
+  push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
```

Comment:
Did we have the WIKI_ACTION_TOKEN yet @byhbt 🤔

Reply:
not yet, i think we need the Repository Admin to set it. 🤔

Reply:
@longnd do you have the Repository Admin for this repo 🤔

or @byhbt can you request the admin access in the `#access-requests Slack channel` so you can have the temporary admin permission for this repo

Reply:
@byhbt do you know the value to set for  `WIKI_ACTION_TOKEN` 😂

Reply:
i think it's `git@github.com:nimblehq/elixir-templates.git`, so here we will set the value `git`

Reply:
Hi, @byhbt could you elaborate on your comment I do not fully understand what is it 🤔

> i think it's git@github.com:nimblehq/elixir-templates.git, so here we will set the value git.

Reply:
This is the code block where `USER_TOKEN` is used. 

```
  - name: Push new wiki content
        run: |
          rsync -av --delete $WIKI_FOLDER/ $TEMP_CLONE_WIKI_FOLDER/ --exclude .git
          cd $TEMP_CLONE_WIKI_FOLDER
          git add .
          git commit -m "Update Wiki content"
          git push -f --set-upstream https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git master
```

So i think the value for the `USER_TOKEN` is `git`. It bases on on the URL to the repository `git@github.com:nimblehq/elixir-templates.git`

Reply:
I have added it @byhbt the value is `git` 

![image](https://user-images.githubusercontent.com/11751745/147198252-018027ba-0e2b-448f-915e-aa178a1a3e19.png)

Should we test this workflow before merge it to develop branch @byhbt

Reply:
adjusted the WIKI_ACTION_TOKEN setting, let test this workflow @byhbt